test/JDBC/expected/fts-contains-vu-prepare.out

```diff
@@ -5051,6 +5051,20 @@ GO
 ~~ROW COUNT: 1~~
 
 
+create table test_special_char_t(id int not null, name text);
+go
+
+insert into test_special_char_t values (1, 'one two three'), (2, 'two three one'), (3, 'one'), (4, 'two'), (5, 'three'), (6, 'one`two'), (7, 'one~two'), (8, 'one!two'),(9, 'one@two'), (10, 'one#two'), (11, 'one$two'),(12, 'one%two'), (13, 'one^two'), (14, 'one&two'), (15, 'one*two'), (16, 'one-two'), (17, 'one_two'), (18, 'one+two'), (19, 'one=two'), (20, 'one\two'), (21, 'one|two'), (22, 'one;two'), (23, 'one:two'), (24, 'one''two'), (25, 'one<two'), (26, 'one>two'), (27, 'one.two'), (28, 'one?two'), (29, 'one/two'), (101, 'one :) two'), (100, 'one @ @ @ @ two') ;
```

Comment:
as indicated ealier, we should test with ore permutations/combinations for both inserted data and search string
special chars (0 or more with spaces) term special chars (0 or more with spaces) term    special chars (0 or more with spaces) term special chars (0 or more with spaces)
with and without ""

Reply:
Done